src/doc/footer.inc

```diff
@@ -5,5 +5,5 @@ or the <a href="http://opensource.org/licenses/MIT">MIT license</a>, at your opt
 </p><p>
 This file may not be copied, modified, or distributed except according to those terms.
 </p></footer>
-<script type="text/javascript" src="jquery.js"></script>
-<script type="text/javascript" src="playpen.js"></script>
+<script type="text/javascript" src="/jquery.js"></script>
+<script type="text/javascript" src="/playpen.js"></script>
```

Comment:
I'm not sure if it's explicitly documented, but that it doesn't require anything else is a source of pride and something many users find useful. Introducing a `python` dependency, however ubiquitous, is a pretty significant downside.